xwiki-pro-macros-ui/src/main/resources/Confluence/Macros/Expand.xml

```diff
@@ -410,9 +410,9 @@ Hello 👀
 #macro (executeMacro)
   $xwiki.ssx.use('Confluence.Macros.Expand')
   #set ($opened = $xcontext.action == 'edit')
-
+  #set ($export = $request.getParameter('format'))
   {{html clean="false" wiki="true"}}
-  &lt;details class="confluence-expand-macro panel panel-default" #if ($opened)open#end&gt;
+  &lt;details class="confluence-expand-macro panel panel-default" #if ($opened || $!export == "html-print")open#end&gt;
```

Comment:
```suggestion
  &lt;details class="confluence-expand-macro panel panel-default" #if ($opened)open#end&gt;
```

Reply:
The Ctrl-P doesn't work, but the PDF export from XWiki works as intended now